src/app/shared/dockstore.model.ts

```diff
@@ -30,11 +32,11 @@ export class Dockstore {
   static readonly GITHUB_CLIENT_ID = 'fill_this_in';
   static readonly GITHUB_AUTH_URL = 'https://github.com/login/oauth/authorize';
 
-  static readonly GITHUB_REDIRECT_URI = Dockstore.LOCAL_URI + '/auth/github.com';
+  static readonly GITHUB_REDIRECT_URI = Dockstore.LOCAL_URI + '/auth/' + Provider.GITHUB;
   static readonly GITHUB_SCOPE = 'read:org,user,user:email';
 
   static readonly QUAYIO_AUTH_URL = 'https://quay.io/oauth/authorize';
-  static readonly QUAYIO_REDIRECT_URI = Dockstore.LOCAL_URI + '/auth/quay';
```

Comment:
Double-checking whether this been tested. This seems to change `Dockstore.LOCAL_URI + '/auth/quay';` and `Dockstore.LOCAL_URI + '/auth/gitlab';` to end with quay.io and gitlab.com respectively.

Reply:
Yes, it's been tested.